R/hook.R

```diff
@@ -13,6 +13,10 @@ common_hook <- function() {
 }
 
 default_load_hook <- function(user_hook = NULL) {
+
+  if (!file.exists(env_file))
+    stop("Unable to find environment file in temporary directory. Try restarting R session.")
```

Comment:
Can you pls break lines be for 75 characters?
```suggestion
    stop(
      "Unable to find environment file in temporary directory.",
      " Try restarting R session."
    )
```